### PR TITLE
Have CocoaPods read some values from package.json

### PR DESCRIPTION
**Pull request by nevir**
Allows you to keep versions, descriptions in sync

### CODE REVIEW

File: React.podspec [hunk 1]
```diff
@@ -1,7 +1,11 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
 Pod::Spec.new do |s|
   s.name                = "React"
-  s.version             = "0.0.1-master"
-  s.summary             = "Build high quality mobile apps using React."
+  s.version             = package['version'].start_with?('0.0.0') ? '0.0.1-master' : package['version']
```
Comment (ide) [inline React.podspec]: We very recently changed (or plan to change) package.json to be version "1000.0.0" in master, so just `package['version']` should suffice

**Comment by ide** [comment]
Thanks, this looks great and should simplify maintenance.

**Comment by greis** [comment]
The version in the podspec file needs to be greater than 0. See this PR https://github.com/facebook/react-native/pull/6381. Is there a problem if the version specified on the package.json changes to `0.0.1-master` instead of `0.0.0-master`?

**Comment by nevir** [comment]
See https://github.com/facebook/react-native/pull/6576#discussion_r56939798

File: React.podspec (hunk 1 above)
Comment (nevir) [inline React.podspec]: Still not in master FYI - but SGTM!

**Comment by ide** [comment]
https://github.com/facebook/react-native/pull/5556 has landed (bumps version to 1000.0.0 in master), let's give this a shot

**Comment by ide** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**